### DISCUSSION TITLE
Aesthetic

### DISCUSSION THREAD

**Discussion by kntsgi** [DiscordDungeons/Bugs#992] [Ideas]
Hello! I have an idea to improve the aesthetics of the experience. idk how the coding side of it works, but it could be really could to update the visuals of the bot so that the HUD is a little more spaced out. maybe use more embeds, images if possible, reactions. also, when it comes to fighting enemies, that page could also have health bars for the characters! that could be cool. maybe for the character, pet and enemy, and have the respective names. it could be cool to eventually be able to name your character at some point. I know these things might be a larger undertaking but it could end up making the user experience that much greater 🔥

**Comment by Sven65** [comment]
We can't do anything about the spacing of the text, as that's an issue on Discords end. As for embeds and images, we are working towards those, slowly but surely.
